pkgs/development/python-modules/annotated-types/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, hatchling
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "annotated-types";
+  version = "0.5.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "annotated-types";
+    repo = "annotated-types";
+    rev = "v${version}";
+    hash = "sha256-zCsWfJ8BQuov8FN+hlm9XBKWAAQ/KHPK/x024A8k2kE=";
+  };
+
+  nativeBuildInputs = [
+    hatchling
+  ];
+
+  propagatedBuildInputs = [
+    typing-extensions
+  ];
+
+  pythonImportsCheck = [ "annotated_types" ];
+
+  meta = with lib; {
+    description = "Reusable constraint types to use with typing.Annotated";
+    homepage = "https://github.com/annotated-types/annotated-types";
```

```suggestion
    homepage = "https://github.com/annotated-types/annotated-types";
    changelog = "https://github.com/annotated-types/annotated-types/releases/tag/${src.rev}";
```
Adding the changelog to the meta is helpful for future updates.

---

Thanks, indeed!